tools/engine_tool/lib/src/commands/fetch_command.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../dependencies.dart';
+import 'command.dart';
+import 'flags.dart';
+
+/// The root 'fetch' command.
+final class FetchCommand extends CommandBase {
+  /// Constructs the 'fetch' command.
+  FetchCommand({
+    required super.environment,
+  }) {
+    argParser.addFlag(
```

Comment:
We should really just have a single `verbose` flag at the runner-level versus at every command that wants one. That does mean configuration will have to be passed down, but that's also fine - this is an intentionally tightly coupled program.

Reply:
Yup that's a good idea, updated!